### PR TITLE
Update SCN section

### PR DESCRIPTION
**Pull request by evtil**
Updated the SCN section of the site. 

Added some information, removed some old.

### CODE REVIEW

File: docs/security-champion/1-new_security_champion.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# I've joined, now what?
+
+## Welcome young padawan
```
Comment (andreabrambilla) [inline docs/security-champion/1-new_security_champion.md:3]: Are we really going wit Star Wars references? Didn't it stop being cool after Disney?

File: docs/security-champion/1-new_security_champion.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+You can also find a calendar of events on [SharePoint](https://statoilsrm.sharepoint.com/sites/securitychampion9)
+
+## Relevant Slack channels
+
+- [#Security-Champions](https://equinor.slack.com/archives/C036HGPBJ04): Your go-to channel for Security Champion events and information
+- [#AppSec](https://equinor.slack.com/archives/CMM6FSW5V): Information regarding AppSec
+
+!!! info
+    [#AppSec](https://equinor.slack.com/archives/CMM6FSW5V) Most general information should be posted here so everyone in Equinor has access to it and can participate!
```
Comment (andreabrambilla) [inline docs/security-champion/1-new_security_champion.md:30]: We could mention that dropping a `@appsecteam` in that channel gets our attention immediately

File: docs/security-champion/1-new_security_champion.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# I've joined, now what?
+
+## Welcome young padawan
+
+This is where the fun begins.
+
+- Join the [#appsec](https://equinor.slack.com/archives/CMM6FSW5V) Slack while you wait for the invitation to the [#security-champion](https://equinor.slack.com/archives/C036HGPBJ04) Slack channel
```
Comment (andreabrambilla) [inline docs/security-champion/1-new_security_champion.md:7]:
```suggestion
- Join the [#appsec](https://equinor.slack.com/archives/CMM6FSW5V) channel on Slack
- You will get an invitation to the [#security-champion](https://equinor.slack.com/archives/C036HGPBJ04) channel within a few days
```

File: docs/security-champion/1-new_security_champion.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- Check out what's listed in our [activities](./2-security_champion_activities.md) section
```
Comment (andreabrambilla) [inline docs/security-champion/1-new_security_champion.md:8]:
```suggestion
- Check out our [activities](./2-security_champion_activities.md)
```

File: docs/security-champion/2-security_champion_activities.md [hunk 5]
```diff
@@ -3,18 +3,27 @@
...
 
-Have a look at our [security requirements](../resources/security_requirements.md) page and define some for your project
+Have a look at our [security requirements](../resources/security_requirements.md) page and define some for your project.
+
+## Check out our guidelines
+
+We have created a few [guidelines](../guidelines/index.md). Please check them out and consider implementing them in your projects where it makes sense.
+
+!!! info
+    Feedback is good, so if you have any, please create a PR at our [github repo](https://github.com/equinor/appsec/)!
```
Comment (andreabrambilla) [inline docs/security-champion/2-security_champion_activities.md:21]: PRs are great but require efforts, let's accept simple nudges as well

```suggestion
    Feedback is good, so if you have any, feel free to contact us, or even create a PR on our [github repo](https://github.com/equinor/appsec/)!
```

File: docs/security-champion/3-faq.md [hunk 6]
```diff
@@ -16,18 +16,33 @@ It depends on what _you_ want to do. It can be everything from just informing th
 
 No, but we highly recommend everyone on sharing. It might also be that you hear about a problem or solution from a team member or co-worker that can be shared. Asking questions is also contributing!
 
+## I don't know anything that's worth sharing
+
+Are you sure? Everyone knows something, and how you apply certain tools or how you've implemented security testing could be very interesting! ``Imposter syndrome`` is real, and we need to combat it.
```
Comment (andreabrambilla) [inline docs/security-champion/3-faq.md:21]:
```suggestion
Are you sure? Everyone knows something, and how you apply certain tools or how you've implemented security testing could be very interesting! The [Imposter syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome) is real, and we need to combat it.
```

File: docs/security-champion/3-faq.md [hunk 7]
```diff
@@ -16,18 +16,33 @@ It depends on what _you_ want to do. It can be everything from just informing th
...
 The Security Champion initiative is a network we invite IT professionals to join and share experiences. Members need to ask their project managers or line leaders for approval to travel and spend time on the network.
 
+## I don't have enough time to spend on security related work
+
+If you feel like the team do not get the needed time to work on security, please reach out to the AppSec team on Slack. We can help convey the importance and help highlight risk in your team.
+
 ## Can we have more Security Champions in our team?
 
-We recommend each team to have 1-2 champions, and share their learnings with the rest of the team. Remember that it is the entire team that is responsible for the security of applications in the team's portfolio. The Security champions will support the team, but not bear any extended responsibility.
+Ideally, each development team should have one or more team-members who takes on the role of Security Champion. When an **area** are supported by many small teams (1-2 persons) this **area** should be represented by one or more security champions.
```
Comment (andreabrambilla) [inline docs/security-champion/3-faq.md:43]: singular/plural mismatch on **area**. Bu to be honest, I don't understand this sentence

File: docs/security-champion/7-about.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+## Activities
+
+Please check our [event](./events/index.md) site for info about upcoming and past Security Champion network events.
+
+The main communication channel for the community is [Slack](https://app.slack.com/client/T02JL00JU/CMM6FSW5V). This is where people can post questions exchange experiences when it comes to different tools and technologies etc.
+
+## Responsibilities
+
+Everyone in the community is responsible for keeping these info sites up to date. They are all based on markdown, don't hesitate to make a PR with fixes, or some new guidelines!
```
Comment (andreabrambilla) [inline docs/security-champion/7-about.md:19]: Is this really everyone responsibility? I'd twist this towards "contributions are welcome" rather than "you are here, now it's your responsibility"

Comment (andreabrambilla) [reply docs/security-champion/7-about.md:19]: Or just drop this paragraph, and keep only the following one

File: docs/security-champion/7-about.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+## Role of the AppSec team
+
+The AppSec team is responsible for driving this network, but this does not mean that you can't contribute!
+
+Would you like to take on some extra responsibilities when it comes to organizing events and keeping there pages up to date? Please reach out to us on [Slack](https://app.slack.com/client/T02JL00JU/CMM6FSW5V) or e-mail at ``appsec[at]equinor.com``
```
Comment (andreabrambilla) [inline docs/security-champion/7-about.md:25]: Possibly misleading. Reading this paragraph as a whole it sounds like, as a security champion, I either let the AppSec team do all the work, or I need to officially sign up (and it is not clear what signing up implies). Can we rephrase this to make it "easier" for people to contribute? 
You have a good idea? Go for it! You want to go big? Get in touch! 🤟

File: docs/security-champion/index.md [hunk 10]
```diff
@@ -1,30 +1,38 @@
...
-A Security Champion is a team-member who amplifies the security message at the team level.
+!!! info
+    You do not need to have any security knowledge to join, but the eagerness to learn and share
+
+## What do we expect from you?
 
-In practice, this means to act as the "security conscience" of the team.
-You do not need to be a security expert to become a Security Champion, an interest for security is more than enough.
+- See something, say something. You are the voice of security, so please use it
+- Join meetings where able
```
Comment (andreabrambilla) [inline docs/security-champion/index.md:13]:
```suggestion
- Join our coffees/seminars/meetups
```

File: docs/security-champion/4-learning-platform.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+- Bragging rights
+- Learn things that may help advance your career!
+
+!!! info
+    You can gain unique merch based on your belt level!
+
+There is a wide arrangement of subjects, and you can do them all if you wish! So there are tracks for:
+
+- AppSec
+- Docker security
```
Comment (steinsiv) [inline docs/security-champion/4-learning-platform.md:21]: Could you add Azure to this list? (one of the advanced tracks was a bit rewarding)

File: docs/security-champion/7-about.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+
+## Contact us
+
+If you are reading this from across the web and want to reach out about the program and how we do things, please do so by sending an e-mail to ``appsec[at]equinor.com`` or reach us through the ``Security Champions Norge`` Slack.
```
Comment (steinsiv) [inline docs/security-champion/7-about.md:29]:
```suggestion
If you are reading this from across the web and want to reach out about the program and how we do things, please do so by sending an e-mail to ``appsec[at]equinor.com`` or reach us through the [Security Champions Norge](https://security-champions-no.slack.com/) Slack.
```

**Review by steinsiv — APPROVED** [review]
Great work! 👍  Made some suggestions you can include if you find them OK. And (I have not tried) but maybe it is possible to --amend the last commit to make it signed to fix that
